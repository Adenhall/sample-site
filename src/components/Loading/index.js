/* @flow */

import React from 'react';

import styles from './styles.scss';

export default () => {
  return (
    <div className={styles.Loading}>
      <p>Loading...</p>
    </div>
  );
};
