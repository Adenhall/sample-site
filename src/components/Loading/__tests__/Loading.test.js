import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';

import Loading from '../index';

describe('<Loading />', () => {
  it('renders', () => {
    const tree = renderer
      .create(
        <MemoryRouter>
          <Loading pastDelay error={false} />
        </MemoryRouter>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
