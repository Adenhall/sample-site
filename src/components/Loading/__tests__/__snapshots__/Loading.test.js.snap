// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading /> renders 1`] = `
<div
  className="Loading"
>
  <p>
    Loading...
  </p>
</div>
`;
